examples/websocket_client.py

```
@@ -0,0 +1,281 @@
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+
+class WandererKillsClient:
+    """WebSocket client for WandererKills real-time killmail subscriptions."""
+    
+    def __init__(self, server_url: str, api_token: str):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)